analyzer/api.go

```diff
@@ -162,43 +162,43 @@ const (
 func (e *Event) String() string {
 	switch *e {
 	case EventStakingTransfer:
-		return "Transfer"
+		return "staking.transfer"
 	case EventStakingBurn:
-		return "Burn"
+		return "staking.burn"
 	case EventStakingAddEscrow:
-		return "AddEscrow"
+		return "staking.escrow.add"
 	case EventStakingTakeEscrow:
-		return "TakeEscrow"
+		return "staking.escrow.take"
 	case EventStakingDebondingStart:
-		return "DebondingStart"
+		return "staking.escrow.debonding_start"
 	case EventStakingReclaimEscrow:
-		return "ReclaimEscrow"
+		return "staking.escrow.reclaim"
 	case EventStakingAllowanceChange:
-		return "AllowanceChange"
+		return "staking.allowance_change"
 	case EventRegistryRuntime:
-		return "Runtime"
+		return "registry.runtime"
 	case EventRegistryEntity:
-		return "Entity"
+		return "registry.entity"
 	case EventRegistryNode:
-		return "Node"
+		return "registry.node"
 	case EventRegistryNodeUnfrozen:
-		return "NodeUnfrozen"
+		return "registry.node_unfrozen"
 	case EventRoothashExecutorCommitted:
-		return "ExecutorCommitted"
-	case EventRoothashDiscrepancyDetected:
-		return "DiscrepancyDetected"
+		return "roothash.executor_committed"
+	case EventRoothashExecutionDiscrepancyDetected:
+		return "roothash.execution_discrepancy_detected"
```

Comment:
https://github.com/oasisprotocol/oasis-core/blob/v22.2.3/go/roothash/api/api.go#L514

you know what's weird, oasis-core actually has an underscore style name for it for json serialization, and that name leaves out the word "detected."

Reply:
Huh, good eye. Omitting "detected" in the indexer too for consistency, thanks.

Fun fact: there's also ["roothash/execution_discrepancy_detected"](https://github.com/oasisprotocol/oasis-core/blob/7f38a068c21160094ce11be1644b08cfb82b53a1/go/roothash/api/api.go#L35-L35), authored by one @pro-wh :)

Reply:
how can he do this to us :laughing: